### PR TITLE
promoting version 6.2.10-31

### PR DESCRIPTION
**Pull request by alon-zada**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.idea
+*bundle.yaml
```
Comment (yuvallevy2) [inline .gitignore:1]: revert

File: bundle.yaml [hunk 2]
```diff
@@ -116,6 +116,12 @@ spec:
       - jsonPath: .metadata.creationTimestamp
         name: Age
         type: date
+      - jsonPath: .status.activeActiveName
```
Comment (yuvallevy2) [inline bundle.yaml:119]: i suggest removing this

File: crds/redb_crd.yaml [hunk 3]
```diff
@@ -39,6 +39,12 @@ spec:
       - jsonPath: .metadata.creationTimestamp
         name: Age
         type: date
+      - jsonPath: .status.activeActiveName
```
Comment (yuvallevy2) [inline crds/redb_crd.yaml:42]: i suggest removing this change

File: google_private_cloud/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+<!-- omit in toc -->
```
Comment (yuvallevy2) [inline google_private_cloud/README.md:1]: delete the entire folder, we are still in preview

File: openshift.bundle.yaml [hunk 5]
```diff
@@ -137,6 +145,12 @@ spec:
       - jsonPath: .metadata.creationTimestamp
         name: Age
         type: date
+      - jsonPath: .status.activeActiveName
```
Comment (yuvallevy2) [inline openshift.bundle.yaml:148]: i suggest that we remove this

File: openshift_3_x/bundle.yaml [hunk 6]
```diff
@@ -6,12 +6,20 @@ metadata:
 allowPrivilegedContainer: false
```
Comment (yuvallevy2) [inline openshift_3_x/bundle.yaml:6]: that folder should have been removed. I will open a ticket to do it next release.

**Review by heinrich-redislabs — APPROVED** [review]

File: redis_on_flash.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
+# Deploying Redis on Flash on K8s using Redis Enterprise operator
```
Comment (yuvallevy2) [inline redis_on_flash.md:1]: this file should be removed (for now)

**Review by heinrich-redislabs — APPROVED** [review]

**Review by yuvallevy2 — APPROVED** [review]
Great!

**Review by heinrich-redislabs — APPROVED** [review]

**Pull request closed without merging**